src/useContentPair.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { createDecoder, createEncoder } from "@waku/core";
+import type { Decoder, Encoder } from "@waku/core/dist/lib/message/version_0";
```

Comment:
Would be good to import the types from the root actually... :)

Reply:
They are not exported from the root unfortunately. Maybe I am missing something?

Reply:
Yes, I mean, maybe we should export them from `@waku/core` root